contents/dataframes_indexing.md

```diff
@@ -232,3 +238,12 @@ s = "subset(salaries(), :salary => ByRow(>(2_000)); skipmissing=true)"
 sco(s; process=without_caption_label)
 ```
 
+```{=comment}
+Rik, we need a example of both filter and subset with multiple conditions, as in:
```

Comment:
That sounds okay. Can we move this comment into an issue? You can assign me to it. I think that TODOs inside code are very difficult to track.

Reply:
updated #54